### ISSUE TITLE
Allow option for alternate default branch name

### ISSUE THREAD

**Issue by ScriptAutomate** [GitJournal/GitJournal#546]
When using GitJournal, if there is a `main` branch instead of a `master` branch (or really any branch other than `master`), it will fail to import the repo. This has resulted in me being unable to use GitJournal on new repos, or with existing repos where the branch name of `master` is unavailable.

During the initial import wizard, there should be a value to provide the name of the default branch or branch of choice. Maybe during the import process itself, it could list available branches? Or before it clones, there could be a field for default branch to use?

```
Version: GitJournal 1.81.0+3076
Platform: android
isPro: false
```

When trying to clone I get these errors, if repo doesn't have a `master` branch:
- `Failed: GitRefNotFound: refs/heads/master`
- `Failed GitException: reference 'refs/heads/master' not found`

**Comment by kevinkleine** [comment]
Same issue on iPad Pro 2018 or 19..

**Comment by vHanda** [comment]
Fixed with 1.82.5
